Pricing.md

```diff
@@ -42,6 +38,13 @@ Please refer to [this document](UsingSeal.md#for-key-server-operators) for detai
     - `Permissioned` mode
         - URL: https://seal-testnet.nodeinfra.com
         - Contact the provider to configure your client and generate a unique key server object id
+- Studio Mirai:
+    - `Open` mode
+        - URL: https://public.key-server.testnet.seal.mirai.cloud
```

Comment:
nit - note that the provider may change the url (by changing it in the onchain object). maybe add somewhere that the url are subject to change, and that the object id is what is needed to be used

Reply:
Good point, done